transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/TransactionOutboxImpl.java

```diff
@@ -149,6 +149,36 @@ public boolean flush(Executor executor) {
         .orElse(false);
   }
 
+  @Override
+  public boolean flushTopic(Executor executor, String topicName) {
+    if (!initialized.get()) {
+      throw new IllegalStateException("Not initialized");
+    }
+    Instant now = clockProvider.get().instant();
+    List<CompletableFuture<Boolean>> futures = new ArrayList<>();
+    futures.add(
```

Comment:
Not needed here. `flushTopic` seems very specific and `expireIdempotencyProtection` is unrelated.